WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -2019,11 +2025,6 @@
 			path = Reader;
 			sourceTree = "<group>";
 		};
-		0C5A1A042D9B080900C25301 /* WordPressAuthenticatorTests */ = {
-			isa = PBXFileSystemSynchronizedRootGroup;
-			path = WordPressAuthenticatorTests;
-			sourceTree = "<group>";
-		};
```

Comment:
Ah! Glad you committed this project change. I got it yesterday too but kept thinking "I'll open another PR then one for this one." I didn't want it out in a PR that did not work on the project file...